examples/webgl_materials_cubemap_mipmaps.html

```diff
@@ -0,0 +1,281 @@
+				customizedCubeTexture.magFilter = THREE.LinearFilter;
+				customizedCubeTexture.format = THREE.RGBAFormat;
+
+				if ( cubeTextures.length > 0 ) {
+
+					customizedCubeTexture.mipmaps = cubeTextures;
+					customizedCubeTexture.generateMipmaps = false;
+
+				}
+
+				customizedCubeTexture.needsUpdate = true;
+
+				return customizedCubeTexture;
+
+			}
+
+			function init() {
+
+				container = document.createElement( 'div' );
+				document.body.appendChild( container );
+
+				camera = new THREE.PerspectiveCamera( 50, window.innerWidth / window.innerHeight, 1, 100000 );
+				camera.position.z = 4000;
+
+				//load cubemap
+
+				scene = new THREE.Scene();
+
+				//lights
+				var ambient = new THREE.AmbientLight( 0xffffff, 0.6 );
+				scene.add( ambient );
+
+				var pendings = [];
+				loadCubeTexture( 'textures/cube/angus/specular_cubemap_256_luv.bin', 256 ).then( function ( reflectCubeTexture ) {
+
+					//rewrite mipmaps
+					var originalCubeTexture = reflectCubeTexture.clone();
+					originalCubeTexture.generateMipmaps = true;
+
+					//models
+					var sphere = new THREE.SphereBufferGeometry( 100, 128, 128 );
+
+					//materials
+					var uniforms = {
+						lodRange: { value: new Float32Array( [ 5, 8 ] ) }, // min & max lod
+						roughness: { value: 0 },
+						specularCubeMap: { value: reflectCubeTexture }
+					};
+
+					var defines = {
+						LUV: 1
+					};
+
+					var material = new THREE.ShaderMaterial( {
+
+						uniforms: Object.assign( {}, uniforms ),
+						vertexShader: document.getElementById( 'vertexShader' ).textContent,
+						fragmentShader: document.getElementById( 'fragmentShader' ).textContent,
+						defines: defines
+
+					} );
+					material.extensions.shaderTextureLOD = true;
+
+					var offset = 250;
+					var originX = -1150;
+					for ( var i = 0; i < 10; ++ i ) {
+
+						//customize mipmaps sphere
+						var roughness = 0.6 * i / 10 + 0.4;
+						var cm = material.clone();
+						cm.uniforms.specularCubeMap.value = originalCubeTexture;
+						cm.uniforms.specularCubeMap.value.needsUpdate = true;
+						cm.uniforms.roughness.value = roughness;
+
+						var c = new THREE.Mesh( sphere, cm );
+						c.position.set( originX + i * offset, 150, 0 );
+						scene.add( c );
+
+						//webgl mipmaps sphere
+						var om = material.clone();
+						om.uniforms.specularCubeMap.value.needsUpdate = true;
+						om.uniforms.roughness.value = roughness;
+
+						var o = new THREE.Mesh( sphere, om );
+						o.position.set( originX + i * offset, -150, 0 );
+						scene.add( o );
+					}
+
+					var backgroundMaterial = material.clone();
+					backgroundMaterial.uniforms.specularCubeMap.value.needsUpdate = true;
+					backgroundMaterial.uniforms.roughness.value = 0.6;
+					backgroundMaterial.side = THREE.DoubleSide;
+
+					backgound = new THREE.Mesh( sphere, backgroundMaterial );
+					backgound.scale.set( 600, 600, 600 );
+					scene.add( backgound );
```

Comment:
IMO, it would also be good to use `Scene.background` and not a custom mesh.

Reply:
removed that